Student-loans (#3885)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/tuition-fee-and-maintenance-loans.md

You do not need to wait until you have a confirmed offer for a teacher training course to apply for student funding. 

## Tuition fee loan
You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course. If you’re eligible for a tuition fee loan, the amount you borrow is paid directly to your course provider.

The maximum tuition fee loan for postgraduate teacher training for a UK applicant is:

* £9,250 per year for a full-time course
* £6,935 per year for a part-time course

The maximum amount you can borrow depends on how much your course costs. 

Tuition fees loans are not means tested. So you do not need to provide evidence of your income or anyone else’s to be eligible to receive one.

## Maintenance loan
You may also be eligible for a maintenance loan to help cover your living costs. 

The maximum amount you may be eligible to borrow is: 

* £8,610, if you live at home with your parents
* £10,227, if you live away from your parents, out of London

> * £13,348, if you live away from your parents, in London

Reviewer: ```suggestion
* £13,348, if you live away from your parents and are studying in London
```

        link_target: "https://www.gov.uk/student-finance-calculator"
navigation: 20.10
navigation_description: Find out which loans you could get to help pay your fees and living expenses while you train.
keywords:
    - Tuition Fee Loan
    - Maintenance Loan
    - Student Loan
    - Student Finance
    - Financial Support
---

Funding is available to help you train to teach. You may be eligible for a student loan to cover the cost of your tuition fees or to help with your living costs. 

You can still apply for a tuition fee loan and a maintenance loan if you already have a student loan, and regardless of whether you get a teaching bursary or scholarship. And you will only have to make loan repayments once you’re earning.

You do not need to wait until you have a confirmed offer for a teacher training course to apply for student funding. 

## Tuition fee loan
You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course. If you’re eligible for a tuition fee loan, the amount you borrow is paid directly to your course provider.

The maximum tuition fee loan for a postgraduate teacher training course for a UK applicant is:

> * £9,250 per year, full time

Reviewer: ```suggestion
* £9,250 per year for a full-time course
```
I think hyphenated is better if it is an adjective - so wonder if rewriting slightly would help (I also think it is slightly confusing without the context of what full-time refers to)

If it helped to reduce repetition, you could perhaps alter the line before to remove `a` and `course`:

The maximum tuition fee loan for postgraduate teacher training for a UK applicant is:

## If you come from outside England

You’ll need to contact your country’s student finance body to find out about your eligibility for funding if you live in Wales, Scotland, or Northern Ireland.

Find out about student finance in:

* [Wales](https://www.studentfinancewales.co.uk/)
* [Scotland](https://www.saas.gov.uk/)
* [Northern Ireland](https://www.studentfinanceni.co.uk/)

Or, you'll need to contact the education authority if you live in Jersey, Guernsey, or the Isle of Man.

Find out about funding in:

* [Jersey](https://www.gov.je/Working/Careers/16To19YearOlds/EnteringHigherEducation/FinancingHigherEducationCourses/FundingDegreeProfessionalQualifications/Pages/index.aspx)
* [Guernsey](https://www.gov.gg/article/152744/Policies)
* [Isle of Man](https://www.gov.im/student-grants)

### Non-UK citizens

If you’re not a UK citizen, you're unlikely to get help funding your training unless you have permission to [live permanently in the UK](https://www.gov.uk/browse/visas-immigration/settle-in-the-uk), for example under the EU Settlement Scheme, or are training to teach languages or physics. You'll also need to meet some other eligibility requirements.

> [Find out about funding for non-UK citizens](/non-uk-teachers/train-to-teach-in-england-as-an-international-student).

Reviewer: ```suggestion
[Find out about funding for non-UK citizens](/non-uk-teachers/fees-and-funding-for-non-uk-trainees).
```
we should change this link?

## Check your eligibility for a bursary or scholarship

You may also be eligible for a tax-free bursary or scholarship, depending on the subject you're training to teach.

> You do not have to pay a bursary or scholarship back. You can get a bursary or scholarship as well as any tuition fee loans and maintenance loans.

Reviewer: ```suggestion
You do not have to pay a bursary or scholarship back. You can get a bursary or scholarship as well as a tuition fee loan and maintenance loan.
```

---
title: "Student finance for teacher training"
heading: "Student finance for teacher training"
description: |-
    Find out about the available student finance for teacher training, including tuition fee loans and maintenance loans to support you while you train.

> related_content:

Reviewer: We link to the Find promo (line 10) at the end of this page which seems slightly odd - better for people to get to the adviser promo?

Author: I'll delete it as we've got the adviser under it.

        link_text: "Student finance calculator"
        link_target: "https://www.gov.uk/student-finance-calculator"
navigation: 20.10
navigation_description: Find out which loans you could get to help pay your fees and living expenses while you train.
keywords:
    - Tuition Fee Loan
    - Maintenance Loan
    - Student Loan
    - Student Finance
    - Financial Support
---

Funding is available to help you train to teach. You may be eligible for a student loan to cover the cost of your tuition fees or to help with your living costs. 

You can still apply for a tuition fee and a maintenance loan if you already have a student loan, and regardless of whether you get a teaching bursary or scholarship. And you will only have to make loan repayments once you’re earning.

You do not need to wait until you have a confirmed offer for a teacher training course to apply for student funding. 

## Tuition fee loans
You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course so you do not have to pay your course fees upfront. If you’re eligible for a tuition fee loan the amount you borrow is paid directly to your course provider.

> The maximum tuition fee loan for a UK applicant:

Reviewer: ```suggestion
The maximum tuition fee loan for a UK applicant is:
```

> * full-time postgraduate teacher training course is £9,250 per year 

Reviewer: ```suggestion
* £9,250 per year for a full-time postgraduate teacher training course
```
would it make more sense to front-load with the number, which also means line 41 would make more sense?

* part-time postgraduate teacher training course is £6,935 per year 

The maximum amount you can borrow depends on how much your course costs. 

Tuition fees loans are not means tested, so you do not need to provide evidence of yours or anyone else’s income to be eligible to receive one.

## Maintenance loans

> You may also be eligible for a maintenance loan to help you cover your living costs. 

Reviewer: ```suggestion
You may also be eligible for a maintenance loan to help cover your living costs. 
```

> The maximum amount you may be eligible to borrow is up to: 

Reviewer: ```suggestion
The maximum amount you may be eligible to borrow is: 
```
Not sure `up to` is needed if we are already saying maximum?

navigation: 20.10
navigation_description: Find out which loans you could get to help pay your fees and living expenses while you train.
keywords:
    - Tuition Fee Loan
    - Maintenance Loan
    - Student Loan
    - Student Finance
    - Financial Support
---

Funding is available to help you train to teach. You may be eligible for a student loan to cover the cost of your tuition fees or to help with your living costs. 

You can still apply for a tuition fee and a maintenance loan if you already have a student loan, and regardless of whether you get a teaching bursary or scholarship. And you will only have to make loan repayments once you’re earning.

You do not need to wait until you have a confirmed offer to apply for student funding. 

## Tuition fee loans

> You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course so you do not have to pay your course fees upfront. If you’re eligible for a tuition fee loan the amount you borrow is paid directly to your course provider.

Author: I'll double check but I think upfront is used in other places. I assumed it's more that they won't have to pay a large amount in one go rather than it being about them paying for the course themselves as ultimately everyone has to pay for it themself.

> You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course so you do not have to pay your course fees upfront. If you’re eligible for a tuition fee loan the amount you borrow is paid directly to your course provider.

Reviewer: ```suggestion
You can apply for a tuition fee loan to cover the cost of your postgraduate teacher training course. If you’re eligible for a tuition fee loan, the amount you borrow is paid directly to your course provider.
```

I wonder whether the upfront part of the sentence could just be removed?

The maximum tuition fee loan for a:

> * full-time postgraduate teacher training course is £9,250 per year for UK applicants

Reviewer 2: Could you move the UK applicants part here to the lead-in line

Or, you'll need to contact the education authority if you live in Jersey, Guernsey, or the Isle of Man.

> Find out about funding in:

Reviewer: ```suggestion
Find out about:
```

> ## Check your eligibility for a scholarship or bursary

Reviewer: We've changed the order of these around the site so it is `bursary or scholarship` as there are more bursaries available. Worth doing on this page whilst we are making updates?

You may also be eligible for a tax-free scholarship or bursary, depending on the subject you're training to teach.

You do not have to pay a scholarship or bursary back. You can receive this as well as any tuition fee and maintenance loans.

> 

Reviewer: We link to the Find promo here at the end of this page which seems slightly odd - better for people to get to the adviser promo?
